docs/index.html

```diff
@@ -46,7 +46,7 @@ <h3>Abstract</h3>
 
 <!--One or two sentences on the motivation behind the problem you are solving. One or two sentences describing the approach you took. One or two sentences on the main result you obtained.-->
 
-Musicians use some kind of synchronization mechanism to keep up with a song. This can be to synchronise other artists or with the timing. Conducting is the art of directing the simultaneous performance of several players or singers by the use of gesture [1]. In order for a machine or a robot to interact with a human player or an orchestra, they need a way to understand a conductor's cues in real-time. One such cue that needs to be understood is the time signature of the piece. We propose to take a subset of time signatures — namely 2/4, 3/4, 4/4, 5/4, 7/8 — and classify these time signatures from a video stream of a conductor performing in real-time.  
+In music, conducting is the art of directing the simultaneous performance of several players or singers by the use of gesture [1]. A conductor provides visual ques which help the orchestra to play in time and provide information about the variations in dynamics of the sound. These visual ques are nothing but various hand gestures. In order for a machine or a robot to interact with a human player and play in an orchestra, they need a way to understand a conductor's cues in real-time. Being able to detect and identify the patterns of hand gestures is quite important. One very common cue that needs to be understood is the time signature of the piece. There are various time signatures that are used for composing music. Each time signature has a different gesture assiciated with it. We propose to take a subset of the time signatures — 2/4, 3/4, 4/4, 5/4, 7/8 — and detect them in a video stream of a conductor performing.  
```

Comment:
Change spelling of "ques" to "cues"
Remove "Being able to detect and identify the patterns of hand gestures is quite important. "

"One very common cue that needs to be understood is the time signature of the piece. There are various time signatures that are used for composing music. Each time signature has a different gesture assiciated with it."
Suggestion:
"One common cue that needs to be understood is the time signature of the piece. The time signature indicates the number of beats in each measure in a piece of music. Conductors use different gestures to coordinate different time signatures."